package.json

```diff
@@ -14,8 +14,10 @@
     "@emotion/core": "^10.0.27",
     "@emotion/styled": "^10.0.27",
     "@guardian/atom-renderer": "1.1.6",
-    "@guardian/automat-client": "^0.2.16",
     "@guardian/consent-management-platform": "4.0.0-8",
+    "@guardian/automat-contributions": "^0.3.5",
+    "@guardian/automat-modules": "^0.3.3",
+    "@guardian/consent-management-platform": "4.0.0-5",
```

Comment:
CMP v4.0.0-5 and -8 are _pre-release_ versions, are you sure it is correct that they are pointing to these?

Reply:
Hey, it's probably a mistake in my rebase against local master. Will bump.

Reply:
No worries!